frontend/jest.config.js

```diff
@@ -9,9 +9,15 @@ const createJestConfig = nextJest({
 // Add any custom config to be passed to Jest
 const customJestConfig = {
   // Add more setup options before each test is run
-  // setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
+  setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
   // if using TypeScript with a baseUrl set to the root directory then you need the below for alias' to work
   moduleDirectories: ['node_modules', '<rootDir>/'],
+  // recommend having this pre-setup as I had pathing issues with tests
+  moduleNameMapper: {
+    '^@/components/(.*)$': '<rootDir>/src/components/$1',
+    '^@/hooks/(.*)$': '<rootDir>/src/hooks/$1',
+    '^@/pages/(.*)$': '<rootDir>/src/pages/$1',
+  },
```

Comment:
It would help to have this setup before interviewees start on the project.
Looks like this will be added by the Next team in the near future ([link](https://github.com/vercel/next.js/blob/canary/examples/with-jest/jest.config.js#L12))